### PR TITLE
Add upload of assets on new batch

### LINKED ISSUE #112: Grab mints events from the queue and upload assets to IPFS when a batch is finished
- [ ] Listen to 'BatchEnd' event and for each token from the finished batch, enqueue the upload of its resources.
- [ ] Create a script connected to the queue, and for each Token create event uploads its resources.

### PR DESCRIPTION
**Pull request by mjlescano**
This PR fixes a the development environment for the keeper package. Where it wasn't able to start using the docker-compose for development. It also refactors the usage of the Ethernauts contract to use the ABI always.

### CODE REVIEW

File: packages/keeper/hardhat.config.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require('@nomiclabs/hardhat-ethers');
 
 module.exports = {
-  solidity: '0.8.4',
+  solidity: '0.8.9',
+  defaultNetwork: 'hardhat',
```
Comment (mariobyn) [inline packages/keeper/hardhat.config.js:5]: defaultNetwork hardhat?

Comment (mjlescano) [reply packages/keeper/hardhat.config.js:5]: Goig to remove the key, that's just the default value, so, there's no need to set it :)

Comment (mjlescano) [reply packages/keeper/hardhat.config.js:5]:
```suggestion
```

**Review by slorenzo — APPROVED** [review]
LGTM

**Pull request merged**